### PR TITLE
Fix perma diff problem during TF import

### PR DESCRIPTION
**Pull request by hao-nan-li**
fixes https://github.com/hashicorp/terraform-provider-google/issues/13515
TLDR:  Resource `groups` has a field `initialGroupConfig` and the default value is "EMPTY". During a TF import, this field is not set to "EMPTY" if it has no value.

If this PR is for Terraform, I acknowledge that I have:

- [ ] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [ ] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [ ] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [ ] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [ ] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:bug
cloudidentity: fixed an issue on `initialGroupConfig` field when users make an import.
```

### CODE REVIEW

**Review by roaks3 — APPROVED** [review]
LGTM

One note on the PR description, it seems like this might fix the issue you referenced. If it does, I think you need to include `fixes` or `closes`.

**Comment by hao-nan-li** [comment]
> LGTM
> 
> One note on the PR description, it seems like this might fix the issue you referenced. If it does, I think you need to include `fixes` or `closes`.

Thanks, will do.

**Comment by dawnbreak** [comment]
I am not sure if this is really solved. I have Terraform 1.5.7 and google provider 5.7.0. I have exactly the same issue that after an import of a `google_cloud_identity_group` the value `initial_group_config` is not set and terraform plan shows this change.

**Comment by roaks3** [comment]
@dawnbreak Could you please open a new issue for the problem your experiencing? You can reference this PR and the original issue there, and that will give us a place to triage/discuss/etc.

**Comment by nathou** [comment]
FYI @dawnbreak the fix seems to only have lasted until google provider version 4.61.0.
Version 4.62.0 has the `initialGroupConfig` setting forcing replacement of group again after import.
